app/views/list_responses/trial_feedbacks.html.slim

```diff
@@ -0,0 +1,57 @@
+= render 'reminder_mailings/section_navigation'
+
+h1 Eingereichte Probezeit Feedbacks
```

Comment:
Then the need_review is no longer needed (along with policy, action, routes etc)